### PR TITLE
[XLSX] Add textRotation to cell style

### PR DESCRIPTION
**Pull request by TekMate8**

### CODE REVIEW

**Comment by Slamdunk** [comment]
Hi, can you add a test please?

**Comment by TekMate8** [comment]
Hi! I'm new at adding pull requests to github repo. I have added some tests to the StyleMergerTest class, I hope it's okay. If not, let me know what else should I do or point me to an example. Thanks.

File: src/Common/Entity/Style/Style.php [hunk 1]
```diff
@@ -383,6 +389,28 @@ public function hasSetWrapText(): bool
...
 
+    public function textRotation(): int
+    {
+        return $this->textRotation;
+    }
+
+    /**
+     * @param int $rotation Rotate text
+     */
+    public function setTextRotation(int $rotation = 0): self
```
Comment (Slamdunk) [inline src/Common/Entity/Style/Style.php]:
```suggestion
    public function setTextRotation(int $rotation): self
```

File: src/Common/Entity/Style/Style.php [hunk 2]
```diff
@@ -383,6 +389,28 @@ public function hasSetWrapText(): bool
...
+    {
+        $this->textRotation = $rotation;
+        $this->hasSetTextRotation = true;
+        $this->isEmpty = false;
```
Comment (Slamdunk) [inline src/Common/Entity/Style/Style.php:404]: Mutation testing tells us this line is not tested, can we remove it?

File: src/Common/Entity/Style/Style.php [hunk 3]
```diff
@@ -383,6 +389,28 @@ public function hasSetWrapText(): bool
...
+
+        return $this;
+    }
+
+    public function hasSetTextRotation(): bool
+    {
+        return $this->hasSetTextRotation;
```
Comment (Slamdunk) [inline src/Common/Entity/Style/Style.php:411]: `0` seems to be the default value: can we remove the `hasSetTextRotation` property and just check if `0 !== $this->textRotation`?

**Review by Slamdunk — CHANGES_REQUESTED** [review]

File: src/Common/Entity/Style/Style.php (hunk 3 above)
Comment (TekMate8) [reply src/Common/Entity/Style/Style.php:411]: Yes.

**Review by Slamdunk — APPROVED** [review]

**Comment by Slamdunk** [comment]
Thank you

**Pull request merged**